src/index.js

```diff
@@ -719,71 +718,14 @@ class LocalstackPlugin {
     // https://github.com/localstack/aws-cdk-local/issues/76#issuecomment-1412590519
     // Issue: https://github.com/localstack/serverless-localstack/issues/125
     if (hostname === "localhost") {
-      try {
-        const port = this.getEdgePort();
-        const options = { host: hostname, port: port };
-        this.checkTCPConnection(options);
-      } catch (e) {
-        const fallbackHostname = "127.0.0.1"
-        this.log.debug(`Reconfiguring hostname to use ${fallbackHostname} (IPv4) because connection to ${hostname} failed`);
-        hostname = fallbackHostname;
-      }
+      hostname = "127.0.0.1";
```

Comment:
I think this solution works but can be considered a breaking change because users have no way of forcing `localhost` anymore and it could potentially break applications depending on this 🤔